apps/design/backend/src/app.test.ts

```diff
@@ -94,11 +94,13 @@ vi.setConfig({
 const vxUser: User = {
   orgId: 'votingworks',
   isVotingWorksUser: true,
+  isSliUser: false,
 };
 
 const nonVxUser: User = {
   orgId: '123',
   isVotingWorksUser: false,
+  isSliUser: false,
```

Comment:
Open to feedback on this approach. Seemed simplest to add this field and while it's not elegant for n+1 orgs, we don't expect to have more than 2 special-case orgs in the near future. Plus a rework of feature flags to send the flags down from the backend rather than these `is*User` bits might make any reorganization in the frontend obsolete soon anyway.

Reply:
That makes sense to me, that the rework of feature flags to send them down from the backend will address this